src/control/cmd/agent/main.go

```diff
@@ -101,11 +145,45 @@ func applyCmdLineOverrides(log logging.Logger, c *client.Configuration, opts *cl
 	}
 }
 
+func setLogOptions(log *logging.LeveledLogger, opts *cliOptions) {
+	if opts.JSONLog {
+		log.WithJSONOutput()
+	}
+
+	switch opts.Debug {
+	case "net":
+		log.WithLogLevel(logging.LogLevelDebug)
+		netdetect.SetLogger(log)
+
+	case "basic":
+		log.WithLogLevel(logging.LogLevelDebug)
+	default:
```

Comment:
Should we do something in the default case? We couldn't hit it at all currently, with the way the option is implemented, so it seems like maybe we should panic, or at least throw some kind of error. Then if someone adds a new choice to the option definition later (or changes the existing ones) and forgets to update here, the failure will be obvious to them.

Reply:
The default: occurs when the -d flag is not specified at all because the string is left empty.  When -d is specified, we will get "basic" if no option or "=basic" is specified, or "net" if "=net" is specified.

Reply:
Oops, right, my bad.